mslib/msui/mscolab.py

```diff
@@ -966,6 +966,34 @@ def delete_account(self, _=None):
             if r.status_code == 200 and json.loads(r.text)["success"] is True:
                 self.logout()
 
+    def editfull_name(self):
+        if verify_user_token(self.mscolab_server_url, self.token):
```

Comment:
there is a newer syntax for this by a decorator. Have a look on the other lines where that is used

Reply:
okkkk